src/charactersheet/viewmodels/dm/encounter_sections/notes_section/form.html

```diff
@@ -0,0 +1,25 @@
+<form data-bind="submit: $component.submit">
+  <div class="card-title h4">
+    <back-button params="{ column: $parents[5] }"></back-button>
```

Comment:
what is this 5?

Reply:
It's just saying that the column is the 5th parent of the current context.

Reply:
I don't really get it. why 5th parent? it feels like a magic number

Reply:
It totally is. That's just the organization of the templates. This template needs to reach up to its 5th parent.